_posts/platform/app/2000-01-01-regions-migration.md

```diff
@@ -6,6 +6,8 @@ tags: app migration region
 index: 17
 ---
 
+Aka *“Project Storky”*.
```

Comment:
Not sure it properly displays like that

Reply:
You can check here: https://documentation-service-pr950.osc-fr1.scalingo.io/platform/app/regions-migration
Seems good to me, no?